README.md

```diff
@@ -156,6 +156,11 @@ Annotate the branches with GitHub PR numbers: <br/>
 git machete github anno-prs
 ```
 
+Check out the PR into local branch, also traverse chain of pull requests upwards, adding branches one by one to git-machete and check them out locally: <br/>
+```shell script
+git machete github checkout-pr [PR_number]
```

Comment:
`[...]` typically denote **optional** params in command-line docs, see `man grep` for a sample

Reply:
So:
```suggestion
git machete github checkout-pr <PR-number>
```